### DISCUSSION TITLE
Compile boost on MacOS

### DISCUSSION THREAD

**Discussion by kernelogic** [filecoin-project/boost#416] [Tutorials]
I tried to compile boost in OSX and after brew install a few dependencies it worked well. Just wanted to share my experience here.

brew install jq
brew install pkg-config
brew install hwloc

**Comment by LexLuthr** [comment]
@kernelogic Thank you. I am also using boost client to make deal from MacOS. It is working without any problems.
I will not down these dependencies and add them to boost docs.

**Comment by nonsense** [comment]
@kernelogic I've updated the README at https://github.com/filecoin-project/boost to include a section on macOS compiling.

**Comment by nonsense** [comment]
@kernelogic I've updated the README at https://github.com/filecoin-project/boost to include a section on macOS compiling.
